helpers/github_helpers.rb

```diff
@@ -5,8 +5,10 @@ def build_contributors(repo)
     return unless config[:github_auth]
     @@github = Github.new do |c|
       c.basic_auth = config[:github_auth]
+      c.connection_options = {headers: {"X-GitHub-OTP" => config[:github_tfa]}}
     end
-    @@repo_contributors = @@github.repos.contributors 'ind9', repo
+    print(", " + repo)
```

Comment:
Yes, it gives a decent idea of progress for few seconds when you're waiting for the page to load.